recipes/elyra-code-viewer-extension/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "elyra-code-viewer-extension" %}
+{% set version = "3.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/elyra-code-viewer-extension-{{ version }}.tar.gz
+  sha256: a3facdb17166334bcb2788b8f02d88cf43d4ce39966cda40b325d6ef234ef518
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - jupyter-packaging
+    - pip
+    - python >=3.7
+  run:
+    - elyra-server ==3.7.0
```

Would it be correct (and better) to say:
```suggestion
    - elyra-server =={{ version }}
```
?